util/OptionsDB.h

```diff
@@ -309,6 +309,14 @@ class FO_COMMON_API OptionsDB {
         m_dirty |= it->second.SetFromValue(value);
     }
 
+    /** Sets the default value of option @p name to the current value */
+    void        OverrideDefault(const std::string& name) {
```

Comment:
Could this have a clearer name? SetCurrentAsDefault or similar?

Reply:
Snuck in on me, no longer used so will remove